src/content/api/loaders.md

```diff
@@ -185,12 +186,12 @@ require('./loader1?xyz!loader2!./resource?rrr');
 
 ### `this.version`
 
-__Loader API version.__ Currently `2`. This is useful for providing backwards compatibility. Using the version you can specify custom logic or fallbacks for breaking changes.
+__Loader API version.__ Currently `2`. This is useful for providing backward compatibility. Using the version you can specify custom logic or fallbacks for breaking changes.
```

Comment:
Why is this necessary?

Reply:
In this context, backward makes more sense.

Reply:
I have read an article on it, it seems both versions are correct. Unless there is more reading to it?

Reply:
Well, what I just figured out is that `backward` is American variant. So, changing it won't create extraordinary improvement. So, reverting it.